script/foundry/utils/DeployHelper.sol

```diff
@@ -318,7 +312,11 @@ contract DeployHelper is Script, BroadcastManager, JsonDeploymentHandler, Storag
                 impl,
                 abi.encodeCall(
                     PILicenseTemplate.initialize,
-                    ("pil", "https://github.com/storyprotocol/protocol-core/blob/main/PIL-Beta-2024-02.pdf")
+                    (
+                        address(protocolAccessManager),
+                        "pil",
+                        "https://github.com/storyprotocol/protocol-core/blob/main/PIL-Beta-2024-02.pdf"
```

Comment:
this one is not working, please update to 
https://github.com/storyprotocol/protocol-core/blob/main/PIL_Beta_Final_2024_02.pdf

Reply:
resolved with 115a1bd